src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
Oh, cool. Well, that's not very clear haha. Every other ponyfill I've seen exposes an alt method. Coolio, will change. So `client.js` is the only place we use `fetch` yeah?